Added concrete example document

michellesanver opened the pull request:
In a meeting that took place the 21st of February 2018 we defined this list as a start to a concrete example document, to specifically list out some things that are unwanted behaviour.

egircys committed:
Adding Code of Conduct

michellesanver committed:
Added concrete example document

sstok commented on contributing/code_of_conduct_concrete_example_document.rst:
```diff
@@ -0,0 +1,16 @@
+Code of Conduct: Concrete Example Document
+==========================================
+This is a living document that serves to give concrete examples to unwanted behaviour. These examples have all taken place somewhere in the PHP community in the past, and are clear code of conduct violations according to the Symfony code of conduct.
```
Documentation lines are usually limited to 72 characters or somewhere near 👍 and headers are followed by a single empty line (sphinxdoc can be picky about this 😄 )

http://symfony.com/doc/current/contributing/documentation/standards.html

michellesanver replied:
Changed :)

sstok commented on contributing/code_of_conduct_concrete_example_document.rst:
```diff
@@ -0,0 +1,16 @@
+Concrete Examples
+-----------------
+- Unwelcome comments regarding a person’s lifestyle choices and practices, including those related to food, health, parenting, drugs, and employment.
+- Deliberate misgendering or use of ‘dead’ (rejected) names
```
Maybe we can add a short explanation about this? Like the one in the referenced source.

michellesanver replied:
@sstok Which explanation are you referring to? I'm afraid I can't find it...

sstok reviewed (changes requested):
Some minor syntax issues. Content looks good 👍

javiereguiluz commented on contributing/code_of_conduct_concrete_example_document.rst:
```diff
@@ -0,0 +1,16 @@
+Concrete Examples
+-----------------
+- Unwelcome comments regarding a person’s lifestyle choices and practices, including those related to food, health, parenting, drugs, and employment.
+- Deliberate misgendering or use of ‘dead’ (rejected) names
```
The `deadname` concept is not that well-known, so what do you think about linking it to https://en.wiktionary.org/wiki/deadname or adding a brief description like this one?

```rst
- Deliberate misgendering or use of deadnames (which are the original names given to
  transgender people before initiating their transition).
```

michellesanver replied:
Changed :)

javiereguiluz reviewed (approved):
It's really sad to read that all this happened for real 😞. Thanks for contributing it!

greg0ire commented on contributing/code_of_conduct_concrete_example_document.rst:
```diff
@@ -0,0 +1,30 @@
+Code of Conduct: Concrete Example Document
+==========================================
+
+This is a living document that serves to give concrete examples to
```
to => of ?

Pierstoval replied:
Not sure "of" is right in this case

GawainLynch replied:
I would say "of" is the correct preposition in this case.

michellesanver replied:
Yeah, of is better. Changed it. Thanks! :)

egircys committed:
add link to examples document

egircys reviewed (commented):
Can it be placed under `contributing/code_of_conduct/concrete_example_document.rst`?

michellesanver commented:
@egircys Of course! Didn't know quite where to place it. I moved it there now 👍

greg0ire commented:
@michellesanver Travis says you should add that document to the toctree (it's unreachable otherwise).

lsmith77 commented:
realtes to https://github.com/symfony/diversity/issues/2

egircys committed:
Change "project's leadership" to "core team"

egircys committed:
Change "race" to "ethnic origin"

egircys committed:
Replace ethnicity with ethnic origin

wouterj commented on contributing/code_of_conduct/concrete_example_document.rst:
```diff
@@ -0,0 +1,30 @@
+- Pattern of inappropriate social contact, such as requesting/assuming
+  inappropriate levels of intimacy with others
+- Continued one-on-one communication after requests to cease
+- Putting down people based on their technology choices or their work
```
Can you fix some minor syntatic things here:

 * Please use `*` as list indicator
 * Please end all entries with a semi-colon, except from the last item which should end with a final stop (`.`).

michellesanver replied:
Thanks! Fixed :)

fabpot commented on contributing/code_of_conduct/concrete_example_document.rst:
```diff
@@ -0,0 +1,30 @@
+- Threats of violence like "The person that created this PR should be
+  punched in the face"
+- Incitement of violence towards any individual, including encouraging a
+  person to commit suicide or to engage in self-harm
```
Would it makes sense to add something like "even as a joke" (or something similar)?

michellesanver replied:
That does make sense, thanks! Added.

fabpot reviewed (commented):
Looks like a great list to me. Thanks.

michellesanver committed:
Updated to say that some jokes are not ok

michellesanver committed:
Syntax

michellesanver committed:
Update concrete_example_document.rst

Commit:
add link to core team

Commit:
change project team to enforcement team

Commit:
update contributor covenant link & documents fix

fabpot committed:
feature #9394 Adding Code of Conduct (egircys)

This PR was submitted for the master branch but it was squashed and merged into the 2.7 branch instead (closes #9394).

Discussion
----------

Adding Code of Conduct

Adding the Contributor Covenant Code of Conduct version 1.4.
Relates to https://github.com/symfony/diversity/issues/1

**TODO**
- [x] Add the link to the enforcement team page
- [x] Add the link to the enforcement process
- [x] Add the link to concrete real-world examples of unwanted behavior
- [x] Change "Project maintainers" to "Enforcement team members"

Part of https://github.com/symfony/symfony-docs/pull/9340 and https://github.com/symfony/symfony-docs/pull/9393

Replaces https://github.com/symfony/symfony/pull/24896

Commits
-------

02e486026 update contributor covenant link & documents fix
2de768f9b change project team to enforcement team
0e7b539a9 add link to core team
e61a403df Replace ethnicity with ethnic origin
2f8aa5d19 Change "race" to "ethnic origin"
1b0e63363 Change "project's leadership" to "core team"
464ea458a add link to examples document
eba3ecfe1 Adding Code of Conduct

fabpot commented:
Thank you very much @michellesanver for taking the time to create the document and everybody for the constructive feedback.
